src/test/run-pass/zst_enum.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![allow(dead_code)]
+
+#[derive(Debug, Copy, Clone)]
+enum X {
+    A = 42 as isize,
+}
+
+fn main() {
+    let x = X::A;
+    let y = x as isize;
+    println!("{:?}", x);
+    println!("{}", y);
```

Comment:
These printlns are not actually tested for, are they? Could a `ui` test check them?

Reply:
Those should simply be an assert